build.sh

```diff
@@ -23,6 +23,11 @@ ARCHS=(
     arm64
     src
 )
+# Architectures we want to guarentee built (currently our build server doesn't build i386)
```

Comment:
Our build server does build i386 for a number of packages, and it is important these never go away or else packages like steam and wine will not install correctly.

Reply:
hmm, some packages that were specified to linux-any haven't been building for i386

Reply:
This is now fixed in https://github.com/pop-os/repo-release/pull/194/commits/23346347031f2a22ae4936d8ea10cdea905953c2

I also added a flag to ignore the binary check https://github.com/pop-os/repo-release/pull/194/commits/0f12649e1cda69f162e494f8f242b7761c3820cb